flake.nix

```diff
@@ -227,15 +227,21 @@
             buildInputs = [ pkgs.python3 ];
           };
 
+          env = {
+            NICKEL_NIX_BUILD_REV = self.shortRev or "dirty";
+          };
+
           buildPackage = { pname, extraBuildArgs ? "", extraArgs ? { } }:
             craneLib.buildPackage ({
               inherit
                 pname
                 src
                 version
-                cargoArtifacts;
+                cargoArtifacts
+                env;
 
               cargoExtraArgs = "${cargoBuildExtraArgs} ${extraBuildArgs} --package ${pname}";
+
```

Comment:
Is this on purpose?

Reply:
The empty line wasn't on purpose. I wonder how that happened...
```suggestion

```